libcontainer/integration/utils_test.go

```diff
@@ -32,6 +32,7 @@ func init() {
 		panic(err)
 	}
 	// Call it to make sure images are downloaded, and to get the paths.
+	// #nosec: G204 (Subprocess launched with variable)
```

Comment:
Thinking if we should just  disable this one in the configuration, because most (all??) would be false positives (or at least: to be ignored)

Reply:
I leave that at the end to you to decide. For me it is also a kind of situation where one better thinks about if it's okay. Especially if not just a value of an arg is coming from a variable, but rather a whole array is passed in. Final call from you?